docs/concepts/budgets.rst

```diff
@@ -27,10 +27,10 @@ Firefly III also features :ref:`categories <categories>`. These are also interes
 The difference between budgets and categories
 ---------------------------------------------
 
-If you can save money every month on a certain subject, it's a budget. Groceries are budget. Bills are a budget. If you travel by train occasionally, it's not a budget.
+If you spend every month on a certain subject, it's a budget. Groceries are budget. Bills are a budget. If you travel by train occasionally, it's not a budget.
```

Comment:
The point is that it's something you can save money on, you can budget it. That's why it's written like this.

Reply:
Please explain more, I don't think I understand you enough.

Reply:
You're right of course that "spending" fits. But the point is that if it's a subject you could save money on, budget on, spend less on, it should be a budget. 

Technically this applies for both categories and budgets because all of those could be subjects that you can save money on. But for budgets the point is that it's something you should actively try to cut back on until it's no longer feasible. 

That's why I wrote "can save money".

Reply:
Maybe "If you try to save money", would fit better in this case?

Reply:
Yes that’s a good one!